CHANGELOG

```diff
@@ -1,11 +1,19 @@
 snowplow-media-player 0.7.0 (2023-xx-xx)
 ---------------------------------------
 ## Summary
-This release adds a more robust unique media identifier. This fixes an issue where duplicate `media_id` values could occur in the media stats table as a result of incorrect tracking implementation (e.g. sharing the same media label across different media types). This release also fixes the incremental materialization of the media_ad_views table by adding a unique primary key.
+This version adds new features powered by a complete refactor of the core processing of the package by moving it out to the new `base` macro functionality provided in `snowplow_utils`. This enables users to now specify custom fields for sessionization and user identification, to add custom entities/SDEs fields to the base events table for redshift/postgres, and to add passthrough fields to the derived tables so you can now more easily add your own fields to our tables.
+
+In addition this release adds a more robust unique media identifier. This fixes an issue where duplicate `media_id` values could occur in the media stats table as a result of incorrect tracking implementation (e.g. sharing the same media label across different media types). This release also fixes the incremental materialization of the media_ad_views table by adding a unique primary key.
+
+## Features
+- Migrate base models to the new `base` macros for flexibility and consistency
+- Add ability to pass fields through to derived media base and ad views tables
+- Add new field `domain_sessionid_array` to derived tables (where applicable)
 
 ## Fixes
```

Comment:
Nit: should we also mention the change in the `session_identifier` logic in the changelog? That we now use the media session ID instead of the domain one?